src/Compilers/CSharp/Portable/Binder/Binder_Statements.cs

```diff
@@ -856,8 +856,7 @@ private TypeSymbolWithAnnotations BindVariableType(CSharpSyntaxNode declarationN
                 // If we got a good result then swap the inferred type for the "var" 
                 if ((object)initializerOpt?.Type != null)
                 {
-                    // Default inferred reference types to a nullable state.
-                    declTypeOpt = TypeSymbolWithAnnotations.Create(initializerOpt.Type, isNullableIfReferenceType: declarator.IsFeatureStaticNullCheckingEnabled());
+                    declTypeOpt = initializerOpt.GetTypeAndNullability(declarator.IsFeatureStaticNullCheckingEnabled());
```

Comment:
Later on, we discussed doing binding with `includeNullability: true`. In the meantime, do you have a test with `var` and the feature disabled that looks at the nullability of the local symbol (I expect oblivious).

Reply:
Good idea. Added.